onchaind/onchaind.c

```diff
@@ -81,6 +81,9 @@ static struct amount_msat our_msat;
 /* Does option_static_remotekey apply to this commitment tx? */
 bool option_static_remotekey;
 
+/* We don't want to log any downstream coin moves if this is a known replay */
+bool open_is_replay;
```

Comment:
This should be "initial_tx_is_replay" perhaps?  open is confusing here...

Reply:
updated in 6c8d7ef94891841dfae0eaeb9b06f8ad142718d4 to be pass through instead of a global (having a global didn't get me much of anything).